src/views/OtherStoresInventoryModal.vue

```diff
@@ -6,25 +6,74 @@
           <ion-icon :icon="close" />
         </ion-button>
       </ion-buttons>
-      <ion-title>{{ translate("Other stores inventory") }}</ion-title>
+      <ion-title>{{ translate("Other stores") }}</ion-title>
+      <ion-buttons slot="end">
+        <ion-button @click="sortByDistance">
+          <ion-icon slot="icon-only" :icon="locationOutline"
+            :color="activeSortType === 'distance' ? 'primary' : 'medium'"></ion-icon>
+        </ion-button>
+        <ion-button @click="sortByName">
+          <ion-icon slot="icon-only" :icon="textOutline"
+            :color="activeSortType === 'name' ? 'primary' : 'medium'"></ion-icon>
+        </ion-button>
+      </ion-buttons>
     </ion-toolbar>
+    <ion-searchbar v-model="queryString" :placeholder="translate('Search store name')" :debounce="500"
+      @ionInput="queryString = $event.target.value; searchFacilities()"
+      @keyup.enter="queryString = $event.target.value; searchFacilities()" />
   </ion-header>
-  <ion-content class="ion-padding">
-    <ion-searchbar v-model="queryString" @keyup.enter="queryString = $event.target.value; searchFacilities()"/>
-    <ion-list v-if="storesInventory.length">
-      <ion-item v-for="storeInventory in storesInventory" :key="storeInventory.facilityName">
-        <ion-label class="ion-text-wrap">{{ storeInventory.facilityName }}</ion-label>
-        <ion-note slot="end">{{ translate('ATP', { count: storeInventory.stock}) }}</ion-note>
+  <ion-content>
+    <ion-list>
+      <ion-item>
+        <ion-label>{{ translate("Hide facilities without stock") }}</ion-label>
+        <ion-toggle @click="toggleHideEmptyStock" />
       </ion-item>
     </ion-list>
+    <ion-accordion-group v-if="atpMappedStoresInventory.length">
+      <ion-accordion v-for="inventory in atpMappedStoresInventory" :key="inventory.storeCode"
+        :value="String(inventory.storeCode)" :toggle-icon="hasWeekCalendar(inventory) ? chevronDownOutline : false"
+        :readonly="!hasWeekCalendar(inventory)">
+        <ion-item slot="header" lines="inset">
+          <ion-label class="ion-text-wrap">
+            <p class="overline">{{ inventory.storeCode }}</p>
+            <h2>{{ inventory.storeName }}</h2>
+            <ion-note v-if="hasWeekCalendar(inventory)" :color="inventory.isOpen ? '' : 'danger'">
+              {{ inventory.hoursDisplay }}
+            </ion-note>
+            <p v-if="typeof inventory.dist === 'number'">{{ Math.round(inventory.dist) }} miles</p>
```

Comment:
Why we have added `typeof` check here?

Reply:
My logic behind adding this typeof check here is to filter out rendering of invalid stores,
while exploring the api (storeLookup) i found out certain stores had a string value "Infinity" in this dist field
This check omits the non number results from being rendered.

![image](https://github.com/user-attachments/assets/fc809b46-d0ae-4b87-93d0-584ca2dfeb1c)

do i need to include these stores too?

Reply:
Yes, this stores should be included without displaying any distance and should be displayed at last.